whereami/whereami_payload.py

```diff
@@ -107,30 +119,30 @@ def call_grpc_backend(backend_service):
 
             return backend_result
 
-        # grab info from GCE metadata
-        try:
-            r = requests.get(METADATA_URL + '?recursive=true',
-                             headers=METADATA_HEADERS)
+        # grab info from cached GCE metadata
+        if len(self.gce_metadata):
```

Comment:
small q : not sure if there exists a failover scenario where the next cluster location is queried instead. If the `gce_metadata` is cached, will it be displaying the outdated `gce_metadata`?